src/ParallelAlgorithms/Events/ObserveTimeStep.hpp

```diff
@@ -50,6 +51,34 @@ using ObserveTimeStep =
     ::Registration::Registrar<Events::ObserveTimeStep, Metavariables>;
 }  // namespace Registrars
 
+namespace {
+struct FormatTimeOutput
+    : tt::ConformsTo<observers::protocols::ReductionDataFormatter> {
+  using reduction_data = Parallel::ReductionData<
```

Comment:
Please move the `ReductionData` from `ObserveTimeStep` into the `detail` namespace above so you can re-use it here. Perhaps name it `ObserveTimeStepReductionData`.